### PR TITLE
Added web route for create and modify

### PR DESCRIPTION
**Pull request by timiajayi**
#142
#143

Corrected
Link : https://todo.zuri.chat/testRoute

### CODE REVIEW

**Comment by kayleb01** [comment]
I'm not sure i understand what this PR is about
please enlighten me

**Comment by timiajayi** [comment]
Routes for create edit and delete for the api.php and web.pho with both post and get and a blade page to test them

File: app/Http/Controllers/TodoController.php [hunk 1]
```diff
@@ -66,6 +66,24 @@ public function showPage()
         return view('test');
     }
 
+    public function edit()
+    {
+        
+return json_encode(['message' => 'Todo created successful'], 200);
```
Comment (kayleb01) [inline app/Http/Controllers/TodoController.php:72]: This doesn't solve any issue

File: resources/views/testRoute.blade.php [hunk 2]
```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
```
Comment (kayleb01) [inline resources/views/testRoute.blade.php:1]: We are not using blade, we are only using Vuejs

**Comment by kayleb01** [comment]
@timiajayi  to be sincere theres no issue like Added web route for create and modify, we are not writing anything web cuz everything the backend is doing in just connecting the zuricore to the apps  FE, so i cannot merge this sorry

**Pull request closed without merging**